packages/panels/resources/views/components/page/index.blade.php

```diff
@@ -32,8 +32,12 @@
             <x-filament-panels::header
                 :actions="$this->getCachedHeaderActions()"
                 :breadcrumbs="filament()->hasBreadcrumbs() ? $this->getBreadcrumbs() : []"
-                :heading="$heading"
-                :subheading="$this->getSubheading()"
+                :attributes="
```

Comment:
```suggestion
                {{-- These are passed through in the bag otherwise Laravel converts View objects to strings prematurely --}}
                :attributes="
```